usage.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: usage
+title: Quarkus Usage Analytics
+subtitle: 
+permalink: /usage/
+---
+
+Starting from Quarkus 3.2, when you run Quarkus for the first time in dev mode (`quarkus dev`, `mvn quarkus:dev`, etc.), you are asked if you agree to contribute anonymous build data to the Quarkus community.
+
+![/assets/images/usage-prompt.png](/assets/images/usage-prompt.png)
+
+If you answer 'Yes', then anonymized data will be sent when you perform a quarkus build,
+to gather usage statistics on how Quarkus is being used and adopted.
+
+This service is provided by Red Hat. More details can be found on the  [usage policy](/usage/policy) page.
```

Comment:
```suggestion
This service is provided by Red Hat. More details can be found in the [usage policy](/usage/policy) page.
```

Reply:
will push fix for this though I thought things written is on the page not in ?